app/validators/poll_validator.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class PollValidator < ActiveModel::Validator
+  MAX_OPTIONS      = 4
+  MAX_OPTION_CHARS = 25
+  MAX_EXPIRATION   = 7.days.freeze
+  MIN_EXPIRATION   = 1.day.freeze
```

Comment:
Same goes for theses, tbh.

Reply:
They make sense to me.

Reply:
The thing is, why I can see why allowing many options may look bad in the UI, I can't see a reason to enforce such limitations on long-running polls or short-running ones.